sass/_chatbox.scss

```diff
@@ -255,6 +255,17 @@
                 width: 100%;
                 resize: none;
             }
+            .chat-textarea-send-button {
+                height: $chat-textarea-height - 35px;
+            }
+            .send-button {
+                position: absolute;
+                bottom: 5px;
+                right: 10px;
```

Comment:
Ok, I was wondering whether that might be the case. Then leave them as pixels but please make them variables.

Then you can for example set the button height to `$chat-textarea-height - $send-button-height - $send-button-margin-bottom`